### ISSUE TITLE
[Pharo 9] Programming via debugger: adding methods to classes from the hierarchy

### ISSUE THREAD

**Issue by dmatveev** [guillep/pharo-core#7038]
_An enhancement is a slight modification of an existing behavior. If you want a new feature use the 
feature request template._

**Describe the request**

Trying out Pharo9 now (the last version I used was 7 I believe). When I send non-existent messages and debugger opens, there's a "Create" button to create a new method in the receiver. Previously (in 7) it allowed to select the class where to create a new method, based on class hierarchy. Now (in 9) it by default creates a new method in the class the instance of which has received the message... This is not what I always would like to do -- is there a way to bring the old behavior back?

**Expected behavior**

Would love to see a way how to introduce methods to classes I select from the inheritance hierarchy -- especially useful when I want to patch my system's standard classes (like String or Number) with extension methods specific to my project.

**Expected development cost**

No idea

**Screenshots**

**Version information:**
 - OS: Ubuntu 18.04
 - Version:
```
Pharo9.0.0
build: 572, commit: 308d3ba
```

**Comment by StephanEggermont** [comment]
+1 This is something I use a lot in DDD, and we have a lot of shallow hierarchies. Having to leave my debugger flow to do a pull up is annoying.

**Comment by StevenCostiou** [comment]
Does that only concern methods or also instance/class variables? @dmatveev @StephanEggermont

**Comment by StephanEggermont** [comment]
That concerns both indeed. Thanks for looking at this systematically

**Comment by StevenCostiou** [comment]
Thanks.
I fixed it for methods (will be available in the next NewTools merge).
For now due to https://github.com/pharo-project/pharo/issues/7083 undeclared variables do not open the debugger and so it is not possible to defined them from the debugger.
It will be resolved when this aforementionned issue is fixed.

I'm reopening the issue unti the merge is done (was closed automatically by github).
